tests/unit/tf/outputs/test_contrastive.py

```diff
@@ -71,6 +71,36 @@ def test_two_tower_constrastive(ecommerce_data: Dataset):
     testing_utils.model_test(model, ecommerce_data)
 
 
+def test_two_tower_constrastive_with_logq_correction(ecommerce_data: Dataset):
+    from merlin.models.tf.transforms.bias import PopularityLogitsCorrection
```

Comment:
Could these imports be moved to the top of the file? My understanding is that the main reason for an inline import inside a function to avoid a circular dependency issue. Otherwise, I think we're following the style of putting all imports at the top of the file so that the imports can be managed by our pre-commit checks.

Reply:
Thank you for clarifying that! I always assumed that if an import is used in one single unit test, we should call it inside the function :)